src/test/java/org/broadinstitute/hellbender/utils/svd/SingularValueDecomposerUnitTest.java

```diff
@@ -56,5 +52,23 @@ public static boolean isUnitaryMatrix(final RealMatrix m){
         }
         return true;
     }
+
+    public void assertSVD(final SVD svd, final RealMatrix m) {
+        final RealMatrix U = svd.getU();
+        final RealMatrix V = svd.getV();
+        final double[] s = svd.getSingularValues();
+
+        final RealMatrix S = MatrixUtils.createRealDiagonalMatrix(svd.getSingularValues());
+        final RealMatrix SPrime = MatrixUtils.inverse(S); //it's diagonal so it's ok to just invert
+
+        Assert.assertEquals(U.multiply(S).multiply(V.transpose()), m);
+        Assert.assertEquals(V.transpose().multiply(V), MatrixUtils.createRealIdentityMatrix(2), "V is unitary");
+        Assert.assertEquals(U.transpose().multiply(U), MatrixUtils.createRealIdentityMatrix(2), "U is unitary");
+
+        Assert.assertTrue(isUnitaryMatrix(U), "matrix U");
+        Assert.assertTrue(isUnitaryMatrix(V), "matrix V");
+        Assert.assertEquals(U.multiply(S).multiply(V.transpose()), m);
+        Assert.assertEquals(V.multiply(SPrime).multiply(U.transpose()), svd.getPinv());
```

Comment:
i added both for good measure. it's test code, performance does not matter as much